ci/k8s/deployment.yml

```diff
@@ -121,6 +121,65 @@ spec:
               key: django-secret-key
         - name: ENABLE_STATS
           value: "yes please"
+      - name: rsr-reports
+        image: eu.gcr.io/akvo-lumen/rsr-backend:${TRAVIS_COMMIT}
+        imagePullPolicy: Always
+        resources:
+          requests:
+            cpu: "50m"
+          limits:
+            cpu: "4000m"
+        ports:
+        - containerPort: 9000
+          protocol: TCP
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 9000
+          initialDelaySeconds: 30
+          timeoutSeconds: 1
+          failureThreshold: 12
+          periodSeconds: 10
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 9000
+          initialDelaySeconds: 30
+          timeoutSeconds: 1
+          failureThreshold: 12
+          periodSeconds: 10
+        volumeMounts:
+            - name: "secret-config"
+              mountPath: "/var/akvo/rsr/code/akvo/settings/42_django.conf"
+              subPath: "django.conf"
+            - name: "secret-config"
+              mountPath: "/rsr/mediasync/rsrleech_id_rsa"
+              subPath: "rsrleech_id_rsa"
+            - name: "secret-config"
+              mountPath: "/rsr/mediasync/media-sync.sh"
+              subPath: "media-sync.sh"
+            - name: "rsr-media-disk"
+              mountPath: "/var/akvo/rsr/mediaroot"
+            - name: "rsr-staticroot-disk"
+              mountPath: "/var/akvo/rsr/staticroot"
+        env:
+        - name: SENTRY_DSN
+          valueFrom:
+            secretKeyRef:
+              name: "rsr-secret"
+              key: sentry-dsn
+        - name: GUNICORN_DEBUG_ARGS
+          valueFrom:
+            secretKeyRef:
+              name: "rsr-secret"
+              key: gunicorn-debug
+        - name: DJANGO_SECRET_KEY
+          valueFrom:
+            secretKeyRef:
+              name: "rsr-secret"
+              key: django-secret-key
+        - name: ENABLE_STATS
```

Comment:
I am missing the env var that tells the container to behave like a report server (same for the other yaml file)

Reply:
Fixed, thanks